packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -194,6 +206,13 @@ const createEntityPageListItems = (
     )}/${entityId}`;
   };
 
+  // Content is knowledge graph
+  const formatContentLink = (uid: string) => {
+    return universe === "prod" || universe === "production"
```

Comment:
Similarly, he created a function to get the yextAccountUrl we should use here as well